src/components/MenuPanel/MenuPanel.js

```diff
@@ -198,6 +200,19 @@ class MenuPanel extends React.PureComponent {
                       {systemApps.map(app => this.renderAppGroup(app))}
                     </AnimDiv>
                   </div>
+                  <Spring
+                    config={springs.smooth}
+                    from={{ opacity: 0 }}
+                    to={{
+                      opacity: window.location.hash.includes('console'),
```

Comment:
Not sure, but `activeInstanceId` may be better (don't remember exactly how it's used), but otherwise we should push the `locator` object down into here instead of relying on `window.location` directly.

There's definitely a better way to do this :).

Reply:
Indeed. :) `activeInstanceId` is the solution.